modules/nw-aws-load-balancer-operator-considerations.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+// * networking/aws_load_balancer_operator/understanding-aws-load-balancer-operator.adoc
+
+:_content-type: REFERENCE
+[id="nw-aws-load-balancer-operator-considerations_{context}"]
+= AWS Load Balancer Operator considerations
+
+Review the following limitations before installing and using the AWS Load Balancer Operator.
+
+* The IP traffic mode only works on AWS Elastic Kubernetes Service (EKS). The AWS Load Balancer Operator disables the IP traffic mode for the AWS Load Balancer Controller and hence, the AWS Load Balancer Controller cannot use the pod readiness gate.
+
+* After installing the AWS Load Balancer Operator, by default, controller command-line flags such as `--disable-ingress-class-annotation` and `--disable-ingress-group-name-annotation` are added to the AWS Load Balancer Controller. Therefore, the AWS Load Balancer Operator does not allow using the `kubernetes.io/ingress.class` and `alb.ingress.kubernetes.io/group.name` annotations in the `AWSLoadBalancerController` custom resource (CR).
```

Comment:
```suggestion
* The AWS Load Balancer Operator  adds command-line flags `--disable-ingress-class-annotation` and `--disable-ingress-group-name-annotation` to the AWS Load Balancer Controller. Therefore, the AWS Load Balancer Operator does not allow using the `kubernetes.io/ingress.class` and `alb.ingress.kubernetes.io/group.name` annotations in the `Ingress` resource.
```
ALBO adds those flags all the time, there is no way to customize it, so I think that `by default` may mislead a little.
The annotations will be applied to `Ingress` resource, sorry, I didn't see it the first time.